app-editors/vscodium/vscodium-1.56.2.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop pax-utils xdg
+
+DESCRIPTION="A community-driven, freely-licensed binary distribution of Microsoft's VSCode"
+HOMEPAGE="https://vscodium.com/"
+SRC_URI="
+	amd64? ( https://github.com/VSCodium/${PN}/releases/download/${PV}/VSCodium-linux-x64-${PV}.tar.gz -> ${P}-amd64.tar.gz )
+	arm? ( https://github.com/VSCodium/${PN}/releases/download/${PV}/VSCodium-linux-armhf-${PV}.tar.gz -> ${P}-arm.tar.gz )
+	arm64? ( https://github.com/VSCodium/${PN}/releases/download/${PV}/VSCodium-linux-arm64-${PV}.tar.gz -> ${P}-arm64.tar.gz )
+"
+
+RESTRICT="mirror strip bindist"
+
+LICENSE="
+	Apache-2.0
+	BSD
+	BSD-1
+	BSD-2
+	BSD-4
+	CC-BY-4.0
+	ISC
+	LGPL-2.1+
+	MIT
+	MPL-2.0
+	openssl
+	PYTHON
+	TextMate-bundle
+	Unlicense
+	UoI-NCSA
+	W3C
+"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~arm ~arm64"
+IUSE=""
+
+RDEPEND="
+	app-accessibility/at-spi2-atk
+	app-crypt/libsecret[crypt]
+	dev-libs/nss
+	media-libs/alsa-lib
+	media-libs/libpng:0/16
+	net-print/cups
+	x11-libs/cairo
+	x11-libs/gtk+:3
+	x11-libs/libnotify
+	x11-libs/libxkbcommon
+	x11-libs/libxkbfile
+	x11-libs/libXScrnSaver
+	x11-libs/libXtst
+	x11-libs/pango
+"
+
+QA_PREBUILT="
+	/opt/vscodium/codium
+	/opt/vscodium/libEGL.so
+	/opt/vscodium/libffmpeg.so
+	/opt/vscodium/libGLESv2.so
+	/opt/vscodium/libvulkan.so*
+	/opt/vscodium/chrome-sandbox
+	/opt/vscodium/libvk_swiftshader.so
+	/opt/vscodium/swiftshader/libEGL.so
+	/opt/vscodium/swiftshader/libGLESv2.so
+	/opt/vscodium/resources/app/extensions/*
+	/opt/vscodium/resources/app/node_modules.asar.unpacked/*
+"
+
+S="${WORKDIR}"
+
+src_install() {
+	# Cleanup
+	rm "${S}/resources/app/LICENSE.txt" || die
+
+	# Install
+	pax-mark m codium
+	insinto "/opt/${PN}"
+	doins -r *
```

Comment:
I think `doins -r ./.` is the preferred way to globbing, or `doins -r .` or something like that.